### DISCUSSION TITLE
New mode TX

### DISCUSSION THREAD

**Discussion by Hunterfnet** [egzumer/uv-k5-firmware-custom#188] [Ideas]
I'm still using the firmware v.019 after try a lot more , and is the best working on my UV5R , i think an extra real TX mode or more ... like DSB (similar to firmware IJV)  will improve a lot for testing purposes , or maybe "unoficial USB and LSB narrowing the portable band and increasing modulation power UP &/or DOWN the tiny portable wave. Thanks

**Comment by egzumer** [comment]
This is not practical whatsover, it doesn't work as it should, I would say even that it is illegal to use. I already added the TX as a function that shoudn't be there, not going further that way. If you want to test, then use the IJV, test, play with it, and come back to my FW for practical usage.

**Comment by f5gkw** [comment]
Hello.
What is illegal to use ?
The DSB ( AM without carrier ) ?

**Reply by egzumer** [reply]
> What is illegal to use ?

Transmitting whith a radio that has thousands times more power on harmonics than on fundamental, with the harmonics being on illegal bands.

**Comment by f5gkw** [comment]
OK  and agreed.
In your message I understood the dsb ;)

**Comment by Hunterfnet** [comment]
Maybe i'm out of wave asking for a simple DSL mode and similar for a "LEGAL firmware modded" section about an Handy totally Legal ( FCC and ECC Rules) . Seems a response about don't make ilegal modded firmwares. But i agree all , included bad days ;))) 
Peace , don't look up ;) thanks
